nservicebus/hosting/critical-errors.md

```diff
@@ -13,7 +13,8 @@ For many scenarios NServiceBus has built-in [error and exception management](/ns
  * An Exception occurs when NServiceBus is attempting to move a message to the Error Queue.
  * There are repeated failures in reading information from a required storage.
  * An exception occurs reading from the input queue.
- * A `IWantToRunWhenBusStartsAndStops.Start` throws an exception.
+ * A `IWantToRunWhenBusStartsAndStops.Start` throws an exception (Version 5 and below).
```

Comment:
I'd change this to:
When `IWantToRunWhenBusStartsAndStops.Start`throws an exception when using Versions 5 and below. 

See Version language here https://github.com/particular/docs.particular.net#version-language